app/controllers/sessions_controller.rb

```diff
@@ -1,13 +1,11 @@
 class SessionsController < ApplicationController
-  def new
-    cookies[:demo_normal] = "normal "
-  end
-
   def create
     user = User.find_by(email: params.dig(:session, :email)&.downcase)
-    if user&.authenticate(params.dig(:session, :password))
+    if user.try(:authenticate, params.dig(:session, :password))
```

Comment:
Góc thắc mắc: Sao đổi &. thành try thế em

Reply:
dạ tại lúc code e đổi giữa &. với try để xem sự khác nhau á chị,với lại thấy tài liệu hướng dẫn sử dụng try an toàn hơn nên em đổi qua try ạ.